core-bundle/src/Security/TwoFactor/BackupCodeManager.php

```diff
@@ -23,6 +23,10 @@ public function isBackupCode($user, string $code): bool
             return false;
         }
 
+        if (null === $user->backupCodes) {
+            return false;
+        }
+
         return \in_array($code, json_decode($user->backupCodes, true), true);
```

Comment:
Should we handle `json_decode` errors (returning `null` value) as well?

Reply:
Fixed in 60836e491.